frontend_csv/src/scanner/mod.rs

```diff
@@ -1,14 +1,43 @@
-mod scanner;
+pub mod scanner;
 /// Scanner implementation!
 pub mod token;
 
 #[cfg(test)]
 mod test {
-    use std::env;
-
+    use crate::scanner::scanner::Scanner;
+    use crate::scanner::token;
+    use std::fs;
     #[test]
     fn scan_simple_one() {
-        let path_file = std::env::vars()["CSV_PATH"];
-        assert!(true);
+        // let path_file = std::env::var("CSV_PATH").unwrap();
+        let path_file = "/home/micky/Desktop/lncodegen.rs/specs/bolt1.csv";
```

Comment:
we should use the os env otherwise the CI fails, and why it is not failing?

Reply:
since the csv file is empty, and all the assertion after readlines.. so it didnt fail....